.github/workflows/backend-tests.yml

```
@@ -37,12 +37,17 @@ jobs:
         run: |
           cd kyros-praxis
           python - << 'PY'
+          import os
           from fastapi.testclient import TestClient
           from services.orchestrator.main import app
           from services.orchestrator.database import SessionLocal
           from services.orchestrator.models import User
           from services.orchestrator.auth import pwd_context
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)